nixos/modules/programs/openvpn3.nix

```diff
@@ -8,11 +8,21 @@ in
 {
   options.programs.openvpn3 = {
     enable = mkEnableOption (lib.mdDoc "the openvpn3 client");
+    package = mkOption {
+      type = types.package;
+      default = pkgs.openvpn3.override {
+        enableSystemdResolved = config.services.resolved.enable;
+      };
+      defaultText = literalExpression "pkgs.openvpn3";
```

Comment:
are you sure that this is a good idea? because if `config.services.resolved.enable` is true the package is a different one and thus search.nixos.org would lie to you so I would suggest to remove this line
```suggestion
```

Reply:
Oh, fair point. Fixed!

Reply:
Actually, removing `defaultText` broke building the manual. I've added it back in https://github.com/NixOS/nixpkgs/pull/242371/commits/caa8139199a480cccaadfa0a1639950457901319, but without lying this time!